index.html

```diff
@@ -29,10 +29,186 @@ <h1 class="content-text">I'm Andrea Glad to see you!</h1>
                     <li class="social-media-icons"><a href="https://chilldigital.com.au"><img src="chill.png" alt=""></a></li>
                 </ul>
             </div>
-        </section>
-      </main>
-    </body>
-</html>
+            </section>
+            <section class="works">
+                <div class="container">
+                  <div class="work-card">
+                  <div>
+                      <img src="SnapshootPortfolio01.png" alt="" />
+                    </div>
```

Comment:
- [ ] Please align the items in each card using flexbox
![image](https://user-images.githubusercontent.com/74744787/185764971-2952af03-0ab7-4836-8eb1-7fb0f88284ac.png)